### ISSUE TITLE
Error: Could not find accessory in dictionary

### ISSUE THREAD

**Issue by mbierman** [basdelfos/homebridge-tuya-web#86]
My log files are full of the following.... any idea what might be wrong?

```
24/2020, 5:44:11 PM] [TuyaWebPlatform] Could not find accessory in dictionary      
[4/24/2020, 5:44:11 PM] [TuyaWebPlatform] Could not find accessory in dictionary      
[4/24/2020, 5:44:11 PM] [TuyaWebPlatform] Could not find accessory in dictionary      
[4/24/2020, 5:44:11 PM] [TuyaWebPlatform] Could not find accessory in dictionary      
[4/24/2020, 5:44:21 PM] [TuyaWebPlatform] Could not find accessory in dictionary      
[4/24/2020, 5:44:21 PM] [TuyaWebPlatform] Could not find accessory in dictionary      
[4/24/2020, 5:44:21 PM] [TuyaWebPlatform] Could not find accessory in dictionary      
[4/24/2020, 5:44:21 PM] [TuyaWebPlatform] Could not find accessory in dictionary      
[4/24/2020, 5:44:21 PM] [TuyaWebPlatform] Could not find accessory in dictionary      
[4/24/2020, 5:44:21 PM] [TuyaWebPlatform] Could not find accessory in dictionary      
[4/24/2020, 5:44:31 PM] [TuyaWebPlatform] Could not find accessory in dictionary      
[4/24/2020, 5:44:31 PM] [TuyaWebPlatform] Could not find accessory in dictionary      
[4/24/2020, 5:44:31 PM] [TuyaWebPlatform] Could not find accessory in dictionary      
[4/24/2020, 5:44:31 PM] [TuyaWebPlatform] Could not find accessory in dictionary      
[4/24/2020, 5:44:31 PM] [TuyaWebPlatform] Could not find accessory in dictionary      
[4/24/2020, 5:44:31 PM] [TuyaWebPlatform] Could not find accessory in dictionary      
[4/24/2020, 5:44:41 PM] [TuyaWebPlatform] Could not find accessory in dictionary      
[4/24/2020, 5:44:41 PM] [TuyaWebPlatform] Could not find accessory in dictionary      
[4/24/2020, 5:44:41 PM] [TuyaWebPlatform] Could not find accessory in dictionary      
[4/24/2020, 5:44:41 PM] [TuyaWebPlatform] Could not find accessory in dictionary      
[4/24/2020, 5:44:41 PM] [TuyaWebPlatform] Could not find accessory in dictionary      
[4/24/2020, 5:44:41 PM] [TuyaWebPlatform] Could not find accessory in dictionary 
```

**Comment by miltykiss** [comment]
In most cases, this is solved by removing all automation/scenes from the Tuya app. (see #34)

**Comment by blaflen** [comment]
Agreed.  I just brought up a new install and had a scene in the app.  I immediately started getting these warnings at polling interval.  If you scroll up to the beginning of the log, you will see an entry like 

> [2020-4-26 23:30:41] [TuyaWebPlatform] Adding: SCENE_NAME (scene / xxxxxxxxxxxxxxxx)                               
> [2020-4-26 23:30:41] [TuyaWebPlatform] Could not init class for device type [scene]                                     

        
This clued me that it was the scene, so I removed it from the app and the warnings stopped (I didn't even need to restart HomeBridge).

**Comment by mykhaliuk** [comment]
> ..so I removed it from the app and the warnings stopped...

Yea, it's a working solution, but not all the people think the scenes are useless and use them as well.

It would be cool to implement the support of the scenes in this plugin. 🙏

**Comment by maelstromchanneled** [comment]
@mykhaliuk, can I ask why you want the scenes in the Tuya app when you have integrated the accessories into HomeKit via HomeBridge?  You could, for example, set up the scenes there?

**Comment by allenlicn** [comment]
@maelstromchanneled many devices are still not supported. I can see scenes as workarounds. For example, tuya RF bridge currently is not supported. If scenes can be supported, one can potentially utilize unsupported devices and other tuya web plugin. On the other hand, if supported devices can be expanded, it will be even better because some devices like sensors can’t get around even with scenes.

**Comment by vascott1** [comment]
I just noticed that I am having this same issue.

[6/8/2020, 22:38:46] [TuyaWebPlatform] Could not find accessory in dictionary (45450869-bc11-4c5d-85a4-0c6e7eddde2a)

I don't remember it happening when I first set it up. I did just add a new device to the TuyaSmart app. Could it be that the device is not supported and this is just normal? The device is a TopGreener TGWF15S WiFi switch. It added to the TuyaSmart app with not a single issue. Should the switch just show up in HomeKit if it will work? I also have a dual plug outdoor outlet that is working just fine with Siri. Not relevant here but the new switch added and works fine with Alexa. 

Simple entries in my config.json

{
            "platform": "TuyaWebPlatform",
            "name": "TuyaWebPlatform",
            "options": {
                "username": "xxxxx@gmail.com",
                "password": "xxxxxx",
                "countryCode": "1",
                "platform": "tuya",
                "pollingInterval": 10
            }
        }

**Comment by maelstromchanneled** [comment]
Have you tried restarting Homebridge?  I've noticed that if I add a device in the Tuya app, then I get this kind of error until I've restarted.  I think it builds the dictionary once at startup ...

**Comment by vascott1** [comment]
exactly what I tried and you posted before me! That seems to have resolved the issue and the new switch is now showing up just fine.

Thanks!
